dev-support/hbase-personality.sh

```diff
@@ -869,6 +869,59 @@ function hbaseanti_patchfile
   return 0
 }
 
+######################################
+
+add_test_type spotless
+
+## @description  only run the test if java or pom changes.
+## @audience     private
+## @stability    evolving
+## @param        filename
+function spotless_filefilter
+{
+  local filename=$1
+
+  if [[ ${filename} =~ \.java$ ]] || [[ ${filename} =~ pom.xml$ ]]; then
```

Comment:
Looking a the `spotless:apply` patch, it included reformatting of markdown, shell scripts, and xml as well. Unless we restrict the spotless scope in the pom configuration, better just let it run always.

Reply:
Ah, the patch was implemented before HBASE-26899, where we only format java and pom.

Let me change this.